modules/__tests__/Media-test.js

```diff
@@ -77,34 +76,34 @@ describe('A <Media>', () => {
     });
 
     describe('and a children element', () => {
-      it('renders its child', () => {
+      it('does not render its child', () => {
         const element = (
           <Media query="">
             <div>hello</div>
           </Media>
         );
 
-        ReactDOM.render(element, node, () => {
-          expect(node.firstChild.innerHTML || '').not.toMatch(/hello/);
+        renderStrict(element, node, () => {
+          expect(node.innerHTML || '').not.toMatch(/hello/);
```

Comment:
`firstChild` was null, which meant that the statement was erroring out. Is this a good enough equivalent, or should it be `node.firstChild && node.firstChild.innerHtml || ''`?

Reply:
I'm a bit confused. Do you mean this test was broken before your changes? Or did this break by adding `renderStrict`?
In any case, I think `node.innerHTML` should be fine.